Package.resolved

```diff
@@ -15,17 +15,17 @@
         "repositoryURL": "https://github.com/amzn/smoke-http.git",
         "state": {
           "branch": null,
-          "revision": "60bc2c4c88700ff46f2cfd1f766c4ac8319488a9",
-          "version": "1.0.0"
+          "revision": "705553c6f59d15f8b9fe2fd458a0ad3755ef9bc4",
+          "version": "1.0.1"
         }
       },
       {
         "package": "swift-log",
         "repositoryURL": "https://github.com/apple/swift-log.git",
         "state": {
           "branch": null,
-          "revision": "f4240bf022a69815241a883c03645444b58ac553",
-          "version": "1.1.0"
+          "revision": "eba9b323b5ba542c119ff17382a4ce737bcdc0b8",
+          "version": "0.0.0"
```

Comment:
Is this change intentional?

Reply:
Strangely version 0.0.0 is correct for Swift 4.2-

https://github.com/apple/swift-log#help-i-need-swift-4

What is actually put in the resolved file depends on if a Swift 4 or 5 compiler was used to auto-generate this file. At build time, the dependency will be changed if what is here isn't compatible with the compiler being used.